wiki/release/release-process.md

```diff
@@ -382,13 +352,13 @@ This is alternative to A. when the user has access to apachepulsar org in hub.do
 
 4. Once it is merged, the chart will be automatically released to Github releases at https://github.com/apache/pulsar-helm-chart and updated to https://pulsar.apache.org/charts.
 
-## 12. Publish Python Clients
+## Publish Python Clients
 
 > **NOTES**
 >
 > 1. You need to create an account on PyPI: https://pypi.org/project/pulsar-client/
```

Comment:
```suggestion
> 1. You need to create an account on PyPI: https://pypi.org/account/register/
```

Reply:
Yes. In addition, the original link shows where the wheels are uploaded, I will add another line to explain it.